regression-tests.dnsdist/test_Caching.py

```diff
@@ -819,6 +819,168 @@ def testCacheExpungeByNameAndType(self):
             total += self._responsesCounter[key]
         self.assertEquals(total, misses)
 
+    def testCacheExpungeBySuffix(self):
+        """
+        Cache: Expunge by name
+
+        """
+        misses = 0
+        ttl = 600
+        name = 'expungebyname.cache.tests.powerdns.com.'
```

Comment:
You are reusing the exact same qname than a previous test in the same suite (`expungebyname.cache.tests.powerdns.com`) so depending on the execution order it might already be cached and the expected miss below will fail. I'd advise using a different qname for each test to prevent this kind of issues.